README.md

```diff
@@ -97,6 +97,41 @@ In cases where this default variable wrapping can cause problems (e.g. a variabl
 
   `{{#wrap}}[{{nowrap link.text}}]({{nowrap link.url}}){{/wrap}}`
 
+#### Handlebars helpers
+
+In addition to the standard helpers which are part of Handlebars, there are a number of additional helpers defined to assist with Ricardian contract specific issues.
+
+___Since 0.0___
+* `wrap` - Takes no parameters. Block level helper to wrap the contents with `<div>` tags for highlighting. See `Variables` section above for more details.
+
+* `nowrap` - Takes a variable. Indicates that the specified variable *should not* be wrapped with `<div>` tags. See `Variables` section above for more details.
+
+* `symbol_to_symbol_code` - Given a variable containing a 'symbol' string, extract and return only the symbol code. For example: '4,EOS' --> returns 'EOS'.
+
+* `asset_to_symbol_code` - Given a variable containing an 'asset' string, extract and return only the symbol code. For example: '2.001 EOS' --> returns 'EOS'.
+
+___Since 0.1___
+* `account_in_permission_level` - Given a variable containing a permission level object, extract and return the account name.
+
+* `permission_in_permission_level` - Given a variable containing a permission level object, extract and return the account name.
```

Comment:
Should say "extract and return the permission name."

Also, it may be good to provide an example like: 'alice@active' ---> returns 'active'.

Reply:
Sure. Just FYI this was built around pulling from authorization structs rather than a string like 'alice@active'. If we DO need something that can handle the string form as just let me know.

Reply:
I misunderstood the nature of `permission_level`. Turns out it isn't a built-in ABI type, which means the `account_in_permission_level` and `permission_in_permission_level` helpers are not actually necessary.

I am not opposed to deprecate those two helpers.